paddle/framework/operator.h

```diff
@@ -110,29 +98,32 @@ class OperatorBase {
   std::shared_ptr<std::unordered_map<std::string, int>> in_out_idxs_;
 };
 
-class KernelContext {
+class OperatorContext {
  public:
-  KernelContext(const OperatorBase* op, const std::shared_ptr<Scope>& scope,
-                const platform::DeviceContext& device_context)
-      : op_(*op), scope_(scope), device_context_(device_context) {}
+  OperatorContext(const OperatorBase* op, const std::shared_ptr<Scope>& scope)
+      : op_(*op), scope_(scope) {}
```

Comment:
Have to check OperatorBase* op not null first.
And we use ```const OperatorBase& op_``` as a member, why not ```const std::shared_ptr<OperatorBase> op_```